### ISSUE TITLE
postcss-html regards entire contents as CSS when the Markdown file has no code blocks

### ISSUE THREAD

**Issue by shinnn** [gucong3000/postcss-html#22]
### Problem

```javascript
const postcss = require('postcss');
const postcssHtml = require('postcss-html');

const valid = postcss().process('# Hi\n```css\na{}\n```\n', {
  syntax: postcssHtml,
  from: 'exmaple.md'
});

valid.error; //=> undefined
```

The code above works fine. But, when `source` contains no code blocks, such as

```js
const invalid = postcss().process('# Hi\n', {
  syntax: postcssHtml,
  from: 'exmaple.md'
});

invalid.error; //=> CssSyntaxError { reason: 'Unknown word', line: 1, column: 1, ...}
```

it tries to parse the entire file contents `# Hi` as CSS and returns a result with `CssSyntaxError`.

### Expected result

Even if the Markdown file doesn't include any code blocks, the PostCSS process should return a result with no errors.

**Comment by gucong3000** [comment]
Thank you for your use and feedback!
Fixed in v0.13.0

**Comment by shinnn** [comment]
@gucong3000 No problem. Thanks for the quick fix and release.
